test/controllers/organization_account_requests_controller_test.rb

```diff
@@ -0,0 +1,45 @@
+require "test_helper"
+
+class OrganizationAccountRequestControllerTest < ActionDispatch::IntegrationTest
+  test "#new" do
+    get new_organization_account_request_path
+
+    assert_response :success
+    assert_select "h1", "New organization account request"
```

Comment:
I think we can get rid of this assertion - it's a personal preference. I started out testing DOM elements heavily in this app and grew to really dislike them because they are so brittle. If anything I think asserting success in the response is sufficient.